eth/tracers/firehose.go

```diff
@@ -247,6 +257,13 @@ func NewFirehose(config *FirehoseConfig) *Firehose {
 		}
 	}
 
+	if config.ConcurrentBlockFlushing {
+		log.Info("Concurrent block flushing enabled: starting goroutine...")
```

Comment:
`goroutine` is too generic, a better comment would have been I think:
```suggestion
		log.Info("Concurrent block flushing enabled: starting block print worker goroutine")
```